electron/package.json

```diff
@@ -31,6 +31,7 @@
         "csv-stringify": "^5.5.3",
         "eiphop": "^1.0.13",
         "electron-context-menu": "2.3.0",
+        "electron-deeplink": "^1.0.6",
```

Comment:
remove the package if its not needed?

Reply:
please also re-run yarn install after removing this so that `electron/yarn.lock` is updated.